src/lib/stores/passage-page.store.ts

```diff
@@ -0,0 +1,70 @@
+import { writable } from 'svelte/store';
+
+export const passagePageMenusObject = writable<PassagePageMenusObject>({
+    showGuideMenu: false,
+    showPassageMenu: false,
+    showMainMenu: false,
+    showLibraryMenu: false,
+});
```

Comment:
Are there cases down the line where multiple menus will be opened at once? Cause my gut reaction here is that this could possibly be an enum and might simplify things elsewhere like the `Object.values(...).some()`.

```suggestion
export enum PassagePageMenu {
  guide = 'guide',
  passage = 'passage',
  main = 'main',
  library = 'library,
}
export const passagePageMenusObject = writable<PassagePageMenu | null>(null);
```

Where null becomes the closed state.